src/CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ add_subdirectory(inference)
 include(cmake/openvino.cmake)
 # preprocessing has dependency on `openvino` for static build
 add_subdirectory(common/preprocessing)
+
+if(CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
Why did you add this additional flag?

Reply:
Actually I am not sure whether we need to add "-Wsign-compare", if not, I will remove it.